coreax/kernel.py

```diff
@@ -348,6 +348,45 @@ def calculate_kernel_matrix_row_sum_mean(
         return self.calculate_kernel_matrix_row_sum(x, max_size) / (1.0 * x.shape[0])
 
 
+class LinearKernel(Kernel):
+    """Define a linear kernel."""
```

Comment:
I know the docstrings for the other kernels are equally sparse, but is there any chance you could add the mathematical definition in here as well? (perhaps also some general comments on the utility of this kernel) It would then be good if you could add these same definitions to the other kernels (in another pull).

Reply:
Added definition in there but left the utility comments off because I'm not aware of any advice around when linear kernel should be used when building coresets vs other kernels.

Reply:
Please can you open an issue to add these descriptions to the other kernel docstrings.